qdrant/grpc_client.go

```diff
@@ -84,3 +93,13 @@ func (c *GrpcClient) Snapshots() SnapshotsClient {
 func (c *GrpcClient) Close() error {
 	return c.conn.Close()
 }
+
+func getClientVersion() (string, error) {
+	packageName := "github.com/qdrant/go-client"
+	cmd := exec.Command("go", "list", "-m", "-f", "{{.Version}}", packageName)
+	output, err := cmd.Output()
+	if err != nil {
+		return "", err
```

Comment:
I think we should swallow the error here are return "Unknown" as the version. Since this is run with the constructor and could be frustrating if it fails somehow.